tests/regression/lp-1898038/task.yaml

```diff
@@ -0,0 +1,33 @@
+summary: docker-support and multipass-support should mount base /etc/apparmor.d
+prepare: |
+    tests.cleanup prepare
+    snap pack test-docker-support-app
+    tests.cleanup defer rm -f test-docker-support-app_1_all.snap
+    snap install --dangerous ./test-docker-support-app_1_all.snap
+    tests.cleanup defer snap remove --purge test-docker-support-app
+    snap pack test-multipass-support-app
+    tests.cleanup defer rm -f test-multipass-support-app_1_all.snap
+    snap install --dangerous ./test-multipass-support-app_1_all.snap
+    tests.cleanup defer snap remove --purge test-multipass-support-app
+execute: |
+    ! aa-status | grep -q 'test-docker-support-test-profile'
```

Comment:
This is sadly incorrect. This will not stop test execution. Please use our `not` helper program instead. There's details inside the source of that program if you want to know why `tests/lib/bin/not`

Reply:
Thanks, fixed in 7c746a1e07